utilities/scan_logs.py

```diff
@@ -68,7 +69,8 @@ def scan_for_errors(log_file_path, error_strings):
             for word in error_strings:
                 # convert the word to lowercase and the line to all lower case
                 # which handles the case where 'warning' will catch 'WARNING' and 'Warning', etc
-                if word.lower() in line.lower():
+                # if word.lower() in line.lower():
+                if re.search(word.lower(), line.lower()):
```

Comment:
This change is to enable scanning for strings with regex, like - "Out of memory: Kill process 1234 (sync_gateway)" as "Out of memory: Kill process \d+ \(sync_gateway\)"